modules/team/variables.tf

```diff
@@ -0,0 +1,13 @@
+variable "team_id" {
+  description = "GitHub team ID"
+  type        = number
+}
+
+variable "members" {
+  description = "GitHub team members with roles"
+  type = list(object({
+    name = string
+    role = string # GitHub team members role (member or maintainer)
```

Comment:
Do we miss option() for role here ?

Reply:
I believe this is intended to be either one or the other (member or maintainer).

Reply:
Actually, according to the local variable, this could also be omitted. In this case, I'd say that the line comment is somewhat misleading.

Reply:
This make no sense to set optional on both levels - module and sub-module

Reply:
Otherwise you have to set defaults on both levels